spec/hako/schedulers/ecs_spec.rb

```diff
@@ -500,4 +501,127 @@
       end
     end
   end
+
+  describe '#oneshot' do
+    context 'when the same task definition exists' do
+      let(:app) { Hako::Application.new(fixture_root.join('jsonnet', 'ecs.jsonnet')) }
+      let(:app_id) { "#{app.id}-oneshot" }
```

Comment:
This isn't app_id but task definition

Reply:
Right. Fixed https://github.com/eagletmt/hako/pull/86/commits/89d6e169cba137cc84c9e29adff720fa5be926f7